src/main/java/core/basesyntax/strategy/TransactionService.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.model.FruitTransaction;
+
+public interface TransactionService {
+    FruitDao dao = new FruitDao();
```

Comment:
Interfarface shouldnt have FruitDao 
```suggestion
    FruitDao dao = new FruitDao();
```